### ISSUE TITLE
PropertyReplacer to support spaces around the brackets

### ISSUE THREAD

**Issue by RobAustin** [OpenHFT/Chronicle-Bytes#146]
The PropertyReplacer looks for 

```
${<token>}
```

the problem is, that if you format the YAML using idea's built-in formatter, then it changes your YAML to 
```
${ <token> }
```

note: the difference is, spaces either side of the `<token>`

This issue is to change the behaviour to accept tokens with starting and ending spaces as the same, in other words, to assume that the following match:
```
${<token>}  ==  ${ <token> }
```

**Comment by glukos** [comment]
The easiest way to achieve this is to consume arbitrary number of leading/trailing brackets inside the spaces.
So, not only 
```
${<token>}  ==  ${ <token> }
```
will be true, but
```
${<token>}  ==  ${ <token> } == ${   <token>             }
```
as well.

I guess nobody intends to use property names with leading/trailing spaces on purpose, so it should be fine.

**Comment by dpisklov** [comment]
I suggest using \s instead of just space. This will cover tabs too for the cases where people use tabs for formatting.

**Comment by glukos** [comment]
@dpisklov Sure, thanks. I've pushed corresponding change

**Comment by hft-team-city** [comment]
Released in [Chronicle-Bytes-2.20.102](https://github.com/OpenHFT/Chronicle-Bytes/releases/chronicle-bytes-2.20.102), [BOM-2.20.142](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.142)
